tests/test_sudoku.py

```diff
@@ -66,31 +70,33 @@ def test_sudoku_solver_hard_task_3():
 
 def test_LEA():
     LEA_1ST_ROW = [
-        {1,2},
-        {1,2},
-        {3,4,5,7,9},
-        {6,8},
-        {1,3,4,6,7,9},
-        {2,3,4,7,8,9},
-        {1,3},
-        {6,8},
-        {3,8}
+        {1, 2},
+        {1, 2},
+        {3, 4, 5, 7, 9},
+        {6, 8},
+        {1, 3, 4, 6, 7, 9},
+        {2, 3, 4, 7, 8, 9},
+        {1, 3},
+        {6, 8},
+        {3, 8}
       ]
 
     LEA_1ST_ROW_SOLUTION = [
-        {1,2},
-        {1,2},
-        {4,7,9},
-        {6,8},
-        {4,7,9},
-        {4,7,9},
-        {1,3},
-        {6,8},
-        {3,8}
+        {1, 2},
+        {1, 2},
+        {4, 7, 9},
+        {6, 8},
+        {4, 7, 9},
+        {4, 7, 9},
+        {1, 3},
+        {6, 8},
+        {3, 8}
       ]
 
-    grid_view = [[_cell_from_alternatives(LEA_1ST_ROW[i]) if j == 0 else Cell() for i in range(9)] for j in range(9)]
+    grid_view = [[_cell_from_alternatives(LEA_1ST_ROW[i]) if (
+        j == 0) else Cell() for i in range(9)] for j in range(9)]
     Sudoku._Sudoku__leave_equal_alternatives(grid_view)
 
-    assert [cell._Cell__alternatives for cell in grid_view[0]] == LEA_1ST_ROW_SOLUTION # проверяем альтернативы в первой строке
-    assert all([grid_view[i][j]._Cell__alternatives == {1, 2, 3, 4, 5, 6, 7, 8, 9} for j in range(9)] for i in range(1, 9))  # проверяем, что альтернативы в других строках не изменлись
+    assert [cell._Cell__alternatives for cell in grid_view[0]] == LEA_1ST_ROW_SOLUTION
+    assert all([grid_view[i][j]._Cell__alternatives == {1, 2,
+           3, 4, 5, 6, 7, 8, 9} for j in range(9)] for i in range(1, 9))
```

Comment:
Опять лишние скобки.
```suggestion
    assert all(
        grid_view[i][j]._Cell__alternatives == {1, 2,3, 4, 5, 6, 7, 8, 9} 
        for j in range(9) for i in range(1, 9)
    )
```